spec/requests/notifications_spec.rb

```diff
@@ -124,7 +132,7 @@
 
     context 'when the user is signed in' do
       before do
-        allow(controller).to receive(:current_user).and_return(user)
+        sign_in user
```

Comment:
Don't need `do...end` here

```suggestion
      before { sign_in user }
```

Reply:
Updated this